test/index.ts

```diff
@@ -0,0 +1,13 @@
+const semver = require('semver')
```

Comment:
Why is this using `require`?

And how about the ones below?

Reply:
updated to import.